cluster-provision/k8s/1.17/provision.sh

```diff
@@ -2,6 +2,24 @@
 
 set -ex
 
+function docker_pull_retry {
```

Comment:
nit: I see mixed indentation in this file, do we have sth like [`shfmt`](https://github.com/mvdan/sh) in place? I guess not? This way we could avoid the discussion about what bracket style etc. we should be using and just use the formatter.

In the meantime we could just make it obligatory by applying it in a presubmit check.

Reply:
in Kubevirt, we have it indeed as you meant i think,
hack/bazel-fmt.sh

i will to align the style manually ? 
or once we integrate shfmt it would do it automatically?
thanks